indico/modules/events/registration/models/registrations.py

```diff
@@ -104,6 +104,14 @@ class Registration(db.Model):
     # relationship backrefs:
     # - registration_form (RegistrationForm.registrations)
 
+    @property
+    def data_by_field(self):
+        return {x.field_data.field_id: x for x in self.data}
+
+    @property
+    def locator(self):
```

Comment:
I've added this locator (in a more powerful version) to upstream, so you can probably get rid of this one when you rebase